schema.d/99-fasschema.ldif

```diff
@@ -23,3 +23,7 @@ objectClasses: ( 1.3.6.1.4.1.30401.1.2.1 NAME 'fasUser' DESC 'FAS user objectcla
 #
 # group extension
 objectClasses: ( 1.3.6.1.4.1.30401.1.2.2 NAME 'fasGroup' DESC 'FAS group objectclass' SUP ipaUserGroup AUXILIARY X-ORIGIN 'Fedora Account System' )
+#
+# agreements
+objectClasses: ( 1.3.6.1.4.1.30401.1.2.3 NAME 'fasAgreement' DESC 'FAS agreement objectclass' X-ORIGIN 'Fedora Account System' )
```

Comment:
Agreements should be a structural object class with mandatory CN/commonName and ipaUniqueID as well as optional ``memberUser`` and ``description``.

```
objectClasses: ( 1.3.6.1.4.1.30401.1.2.3 NAME 'fasAgreement' DESC 'FAS agreement' STRUCTURAL MUST ( ipaUniqueID $ cn ) MAY ( memberUser $ description ) X-ORIGIN 'Fedora Account System' )
```
or maybe use ``SUP ipaAssociation`` instead?